packages/components/socialFeed/NewsFeed/NewsFeed.tsx

```diff
@@ -233,7 +228,11 @@ export const NewsFeed: React.FC<NewsFeedProps> = ({
         }
         keyExtractor={(post) => post.id}
         onScroll={scrollHandler}
-        contentContainerStyle={contentCStyle}
+        contentContainerStyle={{
```

Comment:
This is just personal style but I'd write like this to avoid style undefined. (just my opinion, dont need to change your code)

isMobile ? {
    alignItems: "center",
    width: 100%
} : {
    alignSelf: center,
    width
}